packages/quantic/force-app/main/default/lwc/exampleSearch/resultTemplates/caseResultTemplate.html

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="lgc-bg slds-border_bottom">
+    <lightning-tile type="media">
+      <div class="slds-grid slds-wrap slds-m-vertical_x-small ">
+        <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_3-of-12 slds-text-align_left" style="font-weight: 400; font-size:12px">
```

Comment:
Instead of setting the `style` attribute, could you define a CSS file with reusable classes (e.g., `result__title`, `result__excerpt`, and so on)? This comment applies to all result templates.